tests/utils/test_validation.go

```diff
@@ -65,6 +65,34 @@ func VerifySimpleJsonScanResults(t *testing.T, content string, minViolations, mi
 	}
 }
 
+func VerifyJsonScanTokenValidationResults(t *testing.T, content string, minInactives int) {
+	var results formats.SimpleJsonResults
+	err := json.Unmarshal([]byte(content), &results)
+	if assert.NoError(t, err) {
+		count_inactives := 0
+		for _, result := range results.Secrets {
+			if result.TokenValidation == "Inactive" {
+				count_inactives += 1
+			}
+		}
+		assert.GreaterOrEqual(t, count_inactives, minInactives)
+	}
+}
+
+func VerifySimpleJsonScanTokenValidationResults(t *testing.T, content string, minInactives int) {
+	var results formats.SimpleJsonResults
+	err := json.Unmarshal([]byte(content), &results)
+	if assert.NoError(t, err) {
+		count_inactives := 0
+		for _, result := range results.Secrets {
+			if result.TokenValidation == "Inactive" {
+				count_inactives += 1
+			}
+		}
+		assert.GreaterOrEqual(t, count_inactives, minInactives)
+	}
+}
+
```

Comment:
why only "Incative"? what about the other statuses? why not add it to the already existing VerifySimpleJson funcs?...

Reply:
Only inactive because active requires me to push active tokens which is obviously not possible.